src/lib/markbind/src/preprocessors/componentPreprocessor.js

```diff
@@ -241,16 +240,11 @@ function _preprocessInclude(node, context, config, parser) {
 
   const isIncludeSrcMd = _isHtmlIncludingMarkdown(element, context, filePath);
 
-  const { content, childContext, userDefinedVariables }
-    = parser._renderIncludeFile(actualFilePath, element, context, config, filePath);
-  childContext.source = isIncludeSrcMd ? 'md' : 'html';
-  childContext.callStack.push(context.cwf);
-  parser.extractInnerVariablesIfNotProcessed(content, childContext, config, filePath);
-
-  const innerVariables = parser.getImportedVariableMap(filePath);
-  const fileContent = njUtil.renderRaw(content, {
-    ...userDefinedVariables, ...innerVariables,
-  });
+  const { variablePreprocessor } = config;
+  const {
+    renderedContent,
+    childContext,
+  } = variablePreprocessor.renderIncludeFile(actualFilePath, element, context, filePath);
```

Comment:
Similar boilerplate reduction here, and also the second method of function 4.

The difference here (and also per the original code) is that the child context is returned which is necessary to render nested includes with the parent include's `<variable>`s wrapped by `<include>`s.(https://markbind.org/userGuide/reusingContents.html#specifying-variables-in-an-include)

The 'page' (included content) is also rendered with the extracted variables wrapped inside the include